sdcm/nemesis.py

```diff
@@ -2805,7 +2805,8 @@ def disrupt_memory_stress(self):
             self.target_node.install_epel()
             self.target_node.remoter.sudo('yum install -y stress-ng')
         elif self.target_node.distro.is_ubuntu:
-            self.target_node.remoter.sudo('apt install -y stress-ng')
+            # Install stress on Ubuntu: https://snapcraft.io/install/stress-ng/ubuntu
+            self.target_node.remoter.sudo('snap install stress-ng')
```

Comment:
It seems the apt repo cache isn't updated. The stress-ng can be installed successfully on Ubuntu AMI after updating the apt repo cache.

I have another fix: https://github.com/scylladb/scylla-cluster-tests/pull/3513

Reply:
@amoskong 
According to https://snapcraft.io/install/stress-ng/ubuntu, on Ubuntu 16.4 and later, snap is already installed and ready. So we don't need the `apt` and don't need to update it. Snap it's part of Ubuntu. 
Run `sudo snap install stress-ng` is enough